cms/djangoapps/import_from_modulestore/data.py

```diff
@@ -0,0 +1,23 @@
+"""
+This module contains the data models for the import_from_modulestore app.
+"""
+from django.db.models import TextChoices
+from django.utils.translation import gettext_lazy as _
+
+
+class ImportStatus(TextChoices):
+    """
+    The status of this course import.
+    """
+
+    # PENDING: The import has been created, but the OLX and related data are not yet in the library.
+    # It is not ready to be read.
+    PENDING = 'pending', _('Pending')
+    # READY: The content is staged and ready to be read.
+    READY = 'ready', _('Ready')
+    # IMPORTED: The content has been imported into the library.
+    IMPORTED = 'imported', _('Imported')
+    # CANCELED: The import was canceled before it was imported.
+    CANCELED = 'canceled', _('Canceled')
+    # ERROR: The content could not be imported.
+    ERROR = 'error', _('Error')
```

Comment:
```suggestion
    NOT_STARTED = 'not_started', _('Waiting to stage content')
    STAGING  = 'staging', _('Staging content for import')
    STAGING_FAILED = _('Failed to stage content')
    STAGING_CANCELED = _('Canceled while staging content')
    STAGED = 'staged', _('Content is staged and ready for import')
    IMPORTING = 'importing', _('Importing staged content')
    IMPORTING_FAILED = 'importing_failed', _('Failed to import staged content')
    IMPORTING_CANCELED = 'canceled', _('Canceled while importing staged content')
    IMPORTED = 'imported', _('Successfully imported content')
```
Since the import process happens in two major parts (staging, importing), I think we should reflect that in the statuses.

Also, we can make the human-readable names more informative, which removes the need for code comments, and makes these strings more useful for the end user.

How do these statuses look to you?

Reply:
It seems to me that 2 statuses `STAGING_CANCELED` and `IMPORTING_CANCELED` are redundant, since I can hardly imagine canceling an import that is already in the process of importing, so I left just 1 status for this, and all other changes to the ones you suggested.

Reply:
Sounds good to me.